cime_config/config_grids.xml

```diff
@@ -2293,6 +2323,7 @@
       <file grid="atm|lnd" mask="WCAtl12to45E2r4">$DIN_LOC_ROOT/share/domains/domain.lnd.T62_WCAtl12to45E2r4.210318.nc</file>
       <file grid="atm|lnd" mask="SOwISC12to60E2r4">$DIN_LOC_ROOT/share/domains/domain.lnd.T62_SOwISC12to60E2r4.210119.nc</file>
       <file grid="atm|lnd" mask="ECwISC30to60E2r1">$DIN_LOC_ROOT/share/domains/domain.lnd.T62_ECwISC30to60E2r1.201007.nc</file>
+      <file grid="atm|lnd" mask="ECwISC30to60E3r1">$DIN_LOC_ROOT/share/domains/domain.lnd.T62_ECwISC30to60E3r1.XXXXXX.nc</file>
```

Comment:
@jonbob, I've put what may be unhelpful placeholders throughout this file.  Please edit as you like and push to my branch.